games-rpg/lure: EAPI8 bump, ebuild improvements

mm1ke opened the pull request:
Package-Manager: Portage-3.0.20, Repoman-3.0.3
Signed-off-by: Michael Mair-Keimberger <mmk@levelnine.at>

@ionenwks 

Some notes:
* I've decided to not install the `notes.txt` file since i don't think it's relevant on linux (and makes it easier for the ebuild ;))
* Initially i even had the ebuild without a `src_prepare`. However, in case of `PROTECT.PDF` the file would have been compressed which is why i had to rename it. (i guess it's required for doc files that extension are lowercase?) Anyway, I couldn't find any rules about the file extension need to be lowercase which is why i keep it for the other files.

ionenwks reviewed (commented):
Have to say the original ebuild for this is disgusting. I'm treating this as an entirely new ebuild by you (not that I have much to say beside trivialities given it looks good).

mm1ke commented:
@ionenwks 
Thanks for review. The tip with `newdoc` is perfect. Now the ebuild only contains a `src_install`
I've also updated the `my_l10n` variable. Since it's only used in `src_install` now i made it local and changed the name to lowercase.

mm1ke commented on games-rpg/lure/lure-1.1-r1.ebuild:
```diff
@@ -1,110 +1,59 @@
-LANGS_IUSE="l10n_en l10n_es l10n_de l10n_fr l10n_it"
-IUSE=${LANGS_IUSE}
+IUSE="l10n_de +l10n_en l10n_es l10n_fr l10n_it"
 RESTRICT="mirror"
```
I've also checked the license but also couldn't find a reason for the `RESTRICT=mirror` - i've removed it.

ionenwks reviewed (approved):
> I've also updated the `my_l10n` variable. Since it's only used in `src_install` now i made it local and changed the name to lowercase.

Nice thought :)
